banking-app/test/run.md

```diff
@@ -0,0 +1,63 @@
+# CCF Application Test Environment Setup
+
+## Azure Hosting Environment
+- Azure Confidential Computing VM with [TEE hardware [Azure DC series] (Confidential Linux Image)](https://learn.microsoft.com/en-us/azure/confidential-computing/quick-create-portal)
+    - SGX hardware is installed as TEE (Trusted Execution Environment) 
+    - Both Release and Virtual modes will be supported
+- Or Normal Azure Virtual Machine [(Confidential Linux Image)](https://learn.microsoft.com/en-us/azure/confidential-computing/create-confidential-vm-from-compute-gallery)
+    - No TEE hardware installed 
+    - Only Virtual mode will be supported
+
+## Remote Access your Azure VM using SSH
+```bash
+sudo ssh -i ./key.pem user@VMIP
+```
+## Install CCF runtime on fresh VM
+```bash
+./scripts/setup_ccf_on_vm.sh
+```
+
+### Clone samples repo
+```bash
+# node_ip="10.2.0.4" #VM
+# node_ip="172.17.0.2" #Container
+node_url="https://${node_ip}:8080"
+
+mkdir repos && cd ~/repos
+git clone https://github.com/Aymalla/ccf-app-samples.git
```

Comment:
remember to change this for the original repo in case this PR moves forward.

Reply:
Done